examples/cc26xx/cc26xx-web-demo/cc26xx-web-demo.c

```diff
@@ -966,6 +996,56 @@ PROCESS_THREAD(cc26xx_web_demo_process, ev, data)
 
   PROCESS_END();
 }
+
+PROCESS_THREAD(adc_process, ev, data)
+{
+  PROCESS_BEGIN();
+  static struct etimer et_adc;
+  while(1) {
+    etimer_set(&et_adc, CLOCK_SECOND * 5);
+    PROCESS_WAIT_EVENT();
+    if(etimer_expired(&et_adc)) {
```

Comment:
You can merge those two lines into:
```C
PROCESS_WAIT_EVENT_UNTIL(etimer_expired(&et_adc));
```
Then, the lines below here wouldn't need to be inside of an "`if() {`" block; you could move them two spaces to the left.